js-exercises/animal-sort/animalSort.test.js

```diff
@@ -0,0 +1,41 @@
+import { animalSort } from './animalSort';
```

Comment:
Add tests where either `name` OR `numberOfLegs` property is missing.
Or array is mix of either or both `properties` missing from some items in array.

And then add expected results and test them.

This will help other to know how your program will respond to unexpected input 🙂

Reply:
yes will do.

Reply:
added the test cases @varora1406 in 
3d31f5a